src/components/Filters/Summary.tsx

```diff
@@ -92,22 +93,32 @@ class FilterSummary extends React.Component<
 						</form>
 					</Modal>
 				)}
-				<Flex mb={2}>
-					<Txt
-						fontSize={13}
-						color={this.props.dark ? '#fff' : 'secondary.main'}
-					>
-						Filters ({this.props.filters.length})
-					</Txt>
-
+				<Flex mb={2} justifyContent="space-between">
+					<Flex>
+						<Txt
+							fontSize={13}
```

Comment:
You can remove this, since although in the designs it says `13px`, it actually refers to `fontSize[1]` (which is 13px in the dashboard).